org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -134,10 +173,12 @@ package lang {
 			}
 		} 
 		
+		/** @private */
```

Comment:
Just wondering... if we think having private methods is a good idea - at least at a documentation level - perhaps we should add it to the language